languages/csharp/README.md

```diff
@@ -1,260 +1,51 @@
-  - [Classes](../../concepts/classes.md)
-  - [Objects](../../concepts/objects.md)
-
-### Arithmetic
-
-- Arithmetic overflow
-  - Unsigned integers
-- Signed integers
-- Unsigned integers
-  - Signed integers
-- Floating point numbers
-
-### Methods
-
-- Method overloading
-  - [Classes](../../concepts/classes.md)
-- Named arguments
-- Optional arguments
-- Extension methods (mixin)
-  - Resource lifetime (static/non-static)
+Welcome to the workspace for Exercism v3's C# track!
 
-### General
+This area will contain everything needed to launch the C# track, including:
 
-- [Sameness](../../concepts/sameness.md)
-- [Conditionals](../../concepts/conditionals.md)
-  - [Sameness](../../concepts/sameness.md)
-- [Enumeration](../../concepts/enumeration.md)
-- Iterators (yield)
-  - [Enumeration](../../concepts/enumeration.md)
-- Namespaces
-- [Generics](../../concepts/generics.md)
-- Exception handling
-- Type casting (includes boxing/unboxing)
-- Anonymous types
-  - [Classes](../../concepts/classes.md)
-  - [Anonymous methods](../../concepts/anonymous_functions.md)
-- Concurrency (includes locking and concurrent collections)
-- Asynchronous programming
-- Expressions vs statements
-- Attributes
-- Slicing
-- Locks
-- Unsafe code
-- Implicit/explicit conversion
-- Reflection
+- The new exercises.
+- An updated `config.json` file.
+- Reference documentation that help explain C# concepts for which no appropriate online document could be found.
+- C#-specific documentation for contributors.
 
-## Types
+## Prepartion Status
 
-- [bool](../../types/boolean.md)
-- [string](../../types/string.md)
-- [char](../../types/char.md)
-- object
-- Enum
-- [Null](../../types/null.md)
-- [Nullable type](../../types/nullable.md)
-- `Span<T>`
-- `Memory<T>`
-- [Range](../../types/range.md)
-- Event
-- Delegate
+Before we publicise requesting contribution for this language, the following steps should be done.
 
-### Numeric
+- [ ] [Listed out concepts](../../docs/determining-concepts.md)
+- [ ] [Added GitHub issues for 20 Concept Exercises](../../docs/maintainers/writing-a-concept-exercise-github-issue.md)
+- [ ] Added documentation for contributors
 
-- [int](../../types/integer.md)
-- [uint](../../types/integer.md)
-- [byte](../../types/byte.md)
-- [sbyte](../../types/byte.md)
-- [short](../../types/short.md)
-- [ushort](../../types/short.md)
-- [long](../../types/long.md)
-- [ulong](../../types/long.md)
-- [double](../../types/double.md)
-- [float](../../types/single.md)
-- [decimal](../../types/decimal_number.md)
-- [BigInteger](../../types/big_integer.md)
+## Readiness for Launch
 
-Note: as can be seen, there are many signed/unsigned data type combinations (e.g. `int`/`uint` and `long`/`ulong`). In practice, the unsigned data types are used far less often then the signed ones, so we should probably start with teaching the signed versions. The unsigned versions can then be taught later on.
+Before launch, we need all of the following parts build:
```

Comment:
```suggestion
Before launch, we need all of the following parts to be completed:
```

Some of the things listed are not really "built," such as updating the config.json.